lib/tasks/data/sanitize_data.rake

```diff
@@ -0,0 +1,56 @@
+require 'highline'
+
+namespace :ofn do
+  namespace :data do
+    desc 'Sanitize data'
+    task sanitize: :environment do
+      guard_and_warn
+
+      Spree::User.update_all("email = concat(id, '_ofn_user@example.com'),
+                              login = concat(id, '_ofn_user@example.com'),
+                              unconfirmed_email = concat(id, '_ofn_user@example.com')")
+      Spree::Customer.update_all("email = concat(id, '_ofn_customer@example.com'),
+                                  name = concat('Customer Number ', id)")
+      Spree::Order.update_all("email = concat(id, '_ofn_order@example.com')")
+      Spree::Address.update_all("
+        firstname = concat('Ms. Number', id), lastname = 'Jones',  phone = '01234567890',
+        alternative_phone = '01234567890', address1 = 'Dummy address',
+        address2 = 'Dummy address continuation', city = 'Dummy City', zipcode = '0000',
+        company = null, latitude = null, longitude = null")
+      Spree::TokenizedPermission.update_all("token = null")
+
+      # Sanitize payments related entities
+      Spree::PaymentMethod.update_all("name = concat('Dummy Payment Method', id),
+                                       description = name")
+      Spree::CreditCard.update_all("
+        month = 12, year = 2020, start_month = 12, start_year = 2000,
+        cc_type = 'VISA', first_name = 'Dummy', last_name = 'Dummy', last_digits = '2543'")
+      Spree::Payment.update_all("response_code = null, avs_response = null,
+                                 cvv_response_code = null, identifier = null,
+                                 cvv_response_message = null")
+      Spree::PaypalExpressCheckout.update_all("token = null")
```

Comment:
What about Stripe?

Reply:
it's in the following line: StripeAccount.delete_all
and also included in the Spree::Preference clean up below with the filter "gateway".

Reply:
Oh dear, it was clearly too late for me last night.